src/layer/processing.ts

```diff
@@ -123,14 +130,19 @@ export function createProcessingPayload(
     output: {
       width: params.width,
       height: params.height,
-      responses: [
-        {
-          identifier: 'default',
-          format: {
-            type: params.format,
-          },
-        },
-      ],
+      responses: params.outputResponseId
+        ? [
+            {
+              identifier: params.outputResponseId,
+              format: { type: params.format },
+            },
+          ]
+        : [
+            {
+              identifier: 'default',
+              format: { type: params.format },
+            },
+          ],
```

Comment:
Nitpick: I think this would be more readable:
```javascript
      [
        {
          identifier: params.outputResponseId ? params.outputResponseId : 'default',
          format: { type: params.format },
        },
      ]
```

Fftr.

Reply:
Thanks, totally overlooked that solution. Looks much nicer.

Reply:
Resolving